### DISCUSSION TITLE
Access URL with custom port?

### DISCUSSION THREAD

**Discussion by objectsecurity** [Security-Onion-Solutions/securityonion#7195] [Unsupported Versions] [answered]
Hi!Is there a way to change the port part of the Access URL? If you add the port to the access URL (https://docs.securityonion.net/en/2.3/url-base.html) with colon notation like this: "100.2.3.4:8888", the updating step throws errors. So it will use an external IP or domain correctly but the GUI is only on 443. How can I get this to a different port without having to install another proxy machine? Thank you very much!

**Comment by dougburks** [comment, marked as answer]
I don't believe we support custom ports at this time.

**Reply by objectsecurity** [reply]
Hi Doug, thanks for your response. Security Onion is great btw, used it since the early days in a very small network with no problems. Regarding the port situation, I need 443 for something else and would like to expose this. Maybe a feature suggestion for the future? In the meanwhile we will also look into web rewriting proxies. Best, Ulrich
